### ISSUE TITLE
July meetup

### ISSUE THREAD

**Issue by jakeret** [pydatazrh/meetup#24]

**Comment by jakeret** [comment]
@jherbel could be a potential speaker. He could talk about Approximate Bayesian Inference, which would fit nicely into the bayesian theme.

@betatim what about the location? ETH? Liip?

**Comment by betatim** [comment]
I will ask @uweschmitt our contact at ETH.

**Comment by betatim** [comment]
+ :100:  for a ABC/ABI talk

**Comment by jherbel** [comment]
I will definitely give the talk, it will be about image simulations and abc in the context of cosmology @jakeret # @betatim

**Comment by betatim** [comment]
We have a room 6pm at ETH's main building, room: F33.5. I will try and remember how to find the map that shows where the room is. Thanks @uweschmitt!

**Comment by betatim** [comment]
We announce the meetup then yes?

**Comment by uweschmitt** [comment]
Can we start at 7 pm instead 6 ? I booked the room for an earlier time than needed.

**Comment by betatim** [comment]
Sure, 7pm also works for me.

**Comment by jakeret** [comment]
Yes, lets make it official. @betatim  Do you do the announcement on meetup.com?

Would be cool if there is a third speaker willing to contribute

**Comment by betatim** [comment]
I'll announce it on zee meetup.com. I'll include a "there is one more 15-20min slot left" in the announcement

**Comment by betatim** [comment]
https://www.meetup.com/PyData-Zurich/events/241813543/ - scheduled. @jakeret if you take a quick look for spelling mistakes or edits and find nothing feel free to hit "announce" otherwise I will do it in a few hours.

**Comment by jakeret** [comment]
@jherbel  would you mind sharing your slides? Could you send a Pull Request with the material for `Meetup 6`

**Comment by jherbel** [comment]
I think I did it :stuck_out_tongue_winking_eye:

**Comment by jakeret** [comment]
@betatim can you do the merge? I can't
